src/client/theme/SearchBar/SearchBar.tsx

```diff
@@ -231,6 +237,14 @@ export default function SearchBar({
     };
   }, [isMac, onInputFocus]);
 
+  const onClearSearch = useCallback((event) => {
+    const params = new URLSearchParams(window.location.search);
+    params.delete(SEARCH_PARAM_HIGHLIGHT);
+    history.push(window.location.pathname + `?${params.toString()}`);
+    setInputValue("");
+    search?.autocomplete.setVal("");
```

Comment:
I'm unsure about this change but it was the only way I could make it work as the setInputValue doesn't emit onChange so the autocomplete is not updated otherwise.

Reply:
I will look into it later, to check if there are some unexpected behavior caused by it, since you did some tricks with autocomplete, :)